src/lib/bootloader/grub2_widgets.rb

```diff
@@ -423,6 +423,42 @@ def validate
     end
   end
 
+  # Represents switcher for NVRAM update
+  class UpdateNvramWidget < CWM::CheckBox
+    include Grub2Widget
+
+    def initialize
+      textdomain "bootloader"
+    end
+
+    def label
+      _("Update &NVRAM Entry")
+    end
+
+    def help
+      res = _("<p><b>Update NVRAM</b> will add nvram entry for the bootloader\n" \
+          "in the firmware.\n" \
+          "This is usually desirable unless you want to preserve specific settings\n" \
+          "or need to work around firmware issues.\n" \
+          "\n")
+      res += "</p>"
+
+      res
+    end
+
+    def init
+      self.value = grub2.update_nvram
+    end
+
+    def store
+      grub2.update_nvram = value
+    end
+
+    def validate
```

Comment:
this method is not needed. by default it true.

Reply:
ok